### DISCUSSION TITLE
I always get an exit code

### DISCUSSION THREAD

**Discussion by BaudolinO22** [aclap-dev/video-downloadhelper#407] [Help and Q&A]
### Your question

Hi everybody! 
Since I have updated VDH extension for Mozilla Firefox (125.x) I get always an exit code like the following one:
"Exit code: 1

moz-extension://0d2d60c5-4317-4915-98d8-dd5535c84125/background/main.js:1
receive@moz-extension://0d2d60c5-4317-4915-98d8-dd5535c84125/background/main.js:1:13157
nd</callCatchAppNotFound/</<@moz-extension://0d2d60c5-4317-4915-98d8-dd5535c84125/background/main.js:1:42626"

How I can solve this problem?
More, I add 4 snapshots to let you see
Thanks a lot, Paolo
![Appunti01](https://github.com/aclap-dev/video-downloadhelper/assets/104224395/a5a5686d-efcb-4b56-a333-b4875bb2ec31)
![Appunti02](https://github.com/aclap-dev/video-downloadhelper/assets/104224395/a2089cd2-e387-411a-adb4-8b39454c3b6a)
![Appunti03](https://github.com/aclap-dev/video-downloadhelper/assets/104224395/0433869c-0baf-4fb7-a3c0-6d17e8bbdfbe)
![Appunti04](https://github.com/aclap-dev/video-downloadhelper/assets/104224395/084fcbd2-4db9-4efa-8e0f-e1493ad583ca)

### Addon & CoApp version and details

Video DownloadHelper 8.2.2.8 (stable)
VdhCoApp 2.0.19

**Comment by GrampaWildWilly** [comment]
I am running Windows 11 Pro 64-bit version 23H2 build 22631.3447, Firefox 125.0.1 64-bit, licensed VDH 8.2.2.8a1 beta, CoApp 2.0.19.  I do not work for VDH.  I'm just another user.

I visited your video & this is what VDH told me:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/25a13aba-02f4-4b58-9a1b-fb16b44d8b61)

I told VDH to download the first variant on its menu.  Here is the VDH blue dot status display during the download:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/a6152650-14ee-4648-b800-9c831d2739c1)

This is what VDH downloaded:

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/0daa6d73-e9be-4bc2-bc51-433ba4094ec2)

And here it is playing just fine in VLC.

![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/47cf5d30-d2f8-4c55-a396-3763574080a7)

I don't know what to tell you.  It works for me.

In order to get the VDH beta for Firefox, you can't just get it from the relevant page in the Wiki.  I explain the full process in this discussion: #391.  My explanation is not right at the top of the discussion.  Just look for it.  You'll find it.  It's in there.
